3rd-party-tools/open-fortran-parser-sdf/trans/Makefile

```diff
@@ -0,0 +1,74 @@
+# Usage:
+#
+# If the OFP_HOME environment variable contains the path to the Open Fortran Parser,
+# then this Makefile transforms the coarray Fortran (CAF) source defined in SOURCE 
+# environment variable into the OpenCoarrays CAF (OCAF) dialect printed to the 
+# standard output.  Most commonly, one sets SOURCE on the command line that invokes
+# "make":
+#
+#    make SOURCE=foo.f90 check > ocaf-foo.f90
+#    make clean
+#
+# Alternatively, one can set OFP_HOME also in the "make" invocation: 
+#
+#    make SOURCE=foo.f90 OFP_HOME=/opt/ofp-sdf check >  ocaf-foo.f90
+#    make OFP_HOME=/opt/ofp-sdf clean
+#
+
+ifeq ($(OFP_HOME),)
+  $(error Usage: make SOURCE=foo.f90 OFP_HOME=/path/to/ofp-sdf check >  ocaf-foo.f90)
+endif
+
+include $(OFP_HOME)/make.inc
```

Comment:
This is very tightly coupled with the OFP makefile include, and quite opaque, at least for the time being. It's hard to understand this makefile and all of it's targets without also having the include in front of you. In the current setup this looks like a decent candidate for git submodule or git sub-tree merge. I'll keep my eye on this as it progresses and think about what the best approach is.... maybe contributing a more "normal"/canonical build system upstream to ofp-sdf could help alleviate this. I'll think about it some more.